docs/general_config.md

```diff
@@ -282,6 +286,9 @@ Note: The sound played when `referenceTargetPlayFireSound` is set to `true` is t
 |`playerGravity`     |m/s^2| The graivty vector that impacts the player                                         |
 |`disablePlayerMotionBetweenTrials`|`bool`|Don't allow the player to move when not in a trial?                  |
 |`resetPlayerPositionBetweenTrials`|`bool`|Respawn the player to their original position between trials?        |
+|`headBobEnabled`    |`bool` | Whether or not the headbob effect is enabled.                                    |
+|`headBobAmplitude`  |m      | How high/low the camera Y will go during headBob motion.                         |
```

Comment:
Would be good to note this is additive to the `crouch/playerHeight` and also that it is ± 1/2 the amplitude not ± full amplitude (what this comment implies to me)

Reply:
I have updated the documentation for the headbob. Thank you for pointing these out.